src/components/timeline/TimelineItems.js

```diff
@@ -56,7 +69,6 @@ export default function TimelineItems(props) {
               </Button>
             </Popover>
           </TimelineDetailsExpander>
-
         </Card>
       </div>
     </TimelineItem>
```

Comment:
were any changes made here, or was it just reformatted?

Reply:
It was just reformatting